docs/multicluster/architecture.md

```diff
@@ -32,35 +34,50 @@ interface for resource export/import that can be read/written by all member and
 in the ClusterSet. The Common Area is implemented with a Namespace in the leader cluster for a
 given ClusterSet.
 
+## Antrea Multi-cluster Service
+
+Since Antrea v1.7.0, Multi-cluster Gateways must be configured to support Multi-cluster Service
```

Comment:
This section makes no sense to be here. And readers do not even know what is Multi-cluster Service from this doc so far.

Anyway, let us do some small adjustment in this PR, and revise later. Let us move this section to be after "Antrea Multi-cluster Controller". And in this section, let us first have the "Service Export and Import" sub-section; and add another sub-section called "Service Access Across Clusters", and move this paragraph there.